plugins/woocommerce/lib/packages/Symfony/Component/Intl/Data/Bundle/Compiler/GenrbCompiler.php

```
@@ -0,0 +1,61 @@
+
+    /**
+     * Creates a new compiler based on the "genrb" executable.
+     *
+     * @param string $genrb   Optional. The path to the "genrb" executable
+     * @param string $envVars Optional. Environment variables to be loaded when running "genrb".
+     *
+     * @throws RuntimeException if the "genrb" cannot be found
+     */
+    public function __construct(string $genrb = 'genrb', string $envVars = '')
+    {
+        exec('which '.$genrb, $output, $status);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$output'. (undefined)

(UnusedLocalVariable)